src/DependencyInjection/AnswearOverseasExtension.php

```diff
@@ -18,5 +20,28 @@ public function load(array $configs, ContainerBuilder $container): void
             new FileLocator(__DIR__ . '/../Resources/config')
         );
         $loader->load('services.yaml');
+
+        $configuration = $this->getConfiguration($configs, $container);
+        $config = $this->processConfiguration($configuration, $configs);
+        $this->validateConfig($config);
+
+        $definition = $container->getDefinition(ConfigProvider::class);
+        $definition->setArguments(
+            [
+                EnvironmentEnum::get($config['environment']),
+                $config['apiKey'],
+            ]
+        );
+    }
+
+    private function validateConfig(array $config): void
```

Comment:
You can set validation rules in `Configuration` class: https://symfony.com/doc/current/components/config/definition.html#validation-rules